src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,120 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from './api';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  addNewMovie: (newMovie: Movie) => void;
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+const emptyMovie = {
+  Poster: '',
+  Title: '',
+  Plot: '',
+  imdbID: '',
+};
+
+export const FindMovie: React.FC <Props> = ({ addNewMovie }) => {
+  const [
+    searchQuery,
+    setSearchQuery,
+  ] = useState('');
+
+  const [
+    isErrorOccurred,
+    setIsErrorOccurred,
+  ] = useState(false);
+
+  const [
+    loadedMovie,
+    setLoadedMovie,
+  ] = useState(emptyMovie);
```

Comment:
use `null` as default value

Reply:
Done. Appropriate types and additional conditions (for checking of null value) were added after Type Script crash))